tools/api_proto_plugin/annotations.py

```diff
@@ -10,6 +10,9 @@
 # Page/section titles with special prefixes in the proto comments
 DOC_TITLE_ANNOTATION = 'protodoc-title'
 
+# Specify the type of extensions at extension points
+EXTENSION_TYPE_ANNOTATION = 'protodoc-extension'
```

Comment:
the annotation for extension is just `extension` im wondering if this should be `extension-type` (ie no protodoc prefix)

Reply:
+1

Reply:
updated